src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1

```diff
@@ -1180,19 +1180,22 @@ class Organization: ADOSVTBase
         try {
             $topInactiveUsers = $this.ControlSettings.Organization.TopInactiveUserCount
             $apiURL = "https://vsaex.dev.azure.com/{0}/_apis/UserEntitlements?top={1}&filter=&sortOption=lastAccessDate+ascending&api-version=6.1-preview.3" -f $($this.OrganizationContext.OrganizationName), $topInActiveUsers;
-            $responseObj = [WebRequestHelper]::InvokeGetWebRequest($apiURL);
+            $responseObj = @([WebRequestHelper]::InvokeGetWebRequest($apiURL));
 
             if($responseObj.Count -gt 0)
             {
                 $inactiveUsers =  @()
+                $inactivityThresholdInDays = $this.ControlSettings.Organization.InActiveUserActivityLogsPeriodInDays
```

Comment:
Add below check for InActiveUserActivityLogsPeriodInDays and TopInactiveUserCount

if ([Helpers]::CheckMember($this.ControlSettings.Organization, "InActiveUserActivityLogsPeriodInDays"))